packages/react-components/src/components/ParticipantList.test.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+/* @conditional-compile-remove(rooms) */
+import { mountWithPermissions } from './utils/testUtils';
+/* @conditional-compile-remove(rooms) */
+import { ParticipantList } from './ParticipantList';
+/* @conditional-compile-remove(rooms) */
+import { ParticipantItem } from './ParticipantItem';
+/* @conditional-compile-remove(rooms) */
+import { IContextualMenuItem } from '@fluentui/react';
+/* @conditional-compile-remove(rooms) */
+import { _getPermissions } from '../permissions';
+import { registerIcons } from '@fluentui/react';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+/* @conditional-compile-remove(rooms) */
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+const dummyOnRemoveParticipantCallback = () => {
+  console.log('Removing participant');
+};
+
+describe('ParticipantList tests for different roles', () => {
+  beforeEach(() => {
+    registerIcons({
+      icons: {
+        participantitemoptions: <></>
+      }
+    });
+  });
```

Comment:
To resolve this stable build error:
![image](https://user-images.githubusercontent.com/79475487/185248810-da737275-dcf9-45b2-8616-cd8982946fb7.png)
you can just add a test that tests that ParticipantList should have remove item (with no PermissionProvider as a wrapping component). 
```
test('ParticipantList should have remove item', async () => {
    mount( <ParticipantList
        participants={[{ displayName: 'User1', userId: '1', isRemovable: true }]}
        onRemoveParticipant={dummyOnRemoveParticipantCallback}
      />);
      ...
```